xmake/rules/utils/ispc/ispc.lua

```diff
@@ -0,0 +1,77 @@
+rule("utils.ispc")
+    set_extensions(".ispc")
+    add_deps("utils.inherit.links")
+
+    on_load(function (target)
+        local header_outputdir = path.join(target:autogendir(), "ispc_headers")
+        local obj_outputdir = path.join(target:autogendir(), "ispc_objs")
+        os.mkdir(target:autogendir())
+        os.mkdir(header_outputdir)
+        os.mkdir(obj_outputdir)
+        target:add("includedirs", header_outputdir, {public = true})
+    end)
+    before_buildcmd_file(function (target, batchcmds, sourcefile_ispc, opt)
+        import("lib.detect.find_tool")
+        ispc = find_tool("ispc")
+        assert(ispc, "ispc not found!")
```

Comment:
missing local keyword

`local ispc = assert(find_tool("ispc"), "ispc not found!")`